routing_model/index.html

```diff
@@ -684,7 +707,7 @@ <h2>Process Adult/Elder promotion and demotion needed from last check</h2>
       OneVoteConsensused --> WaitComplete
       WaitComplete(("Check?"))
       WaitComplete--"WAITED_VOTES<br/>.is_empty()<br/>(Wait complete)"-->MarkNewElderAdults
-      MarkNewElderAdults["For all nodes:<br/>set_elder_status(node, is_elder)"]
+      MarkNewElderAdults["for node in nodes {<br/>is_elder = new_section_info.contains(node)<br/>set_elder_status(node, is_elder)<br/>}"]
```

Comment:
You don't have access to nodes.
Maybe you'd be better off creating a function you want:
`update_elder_status(new_section_info)`